2333/2/libs/rapper.rb

```diff
@@ -0,0 +1,35 @@
+require 'russian_obscenity'
```

Comment:
It's a very good practice to use Gemfile in your projects.  But creating a Gemfile in your project is not sufficient.

Right now when you run `bundle install` it installs gems in your home directory and then you require those global gems in your app. Also here you require some gems that are not listed in Gemfile, like `terminal-table` and `russian_obscenity`.

Instead, we want to make sure that in our app we use versions of gems that are specified in `Gemfile.lock`. In order to do this, we should init Bundler in our app.

There are 2 ways to do this:
1. Use [`Bundler.setup`](https://bundler.io/v1.5/bundler_setup.html) and then continue to use `require`.
2. Use [`Bundler.require`](https://bundler.io/v1.5/groups.html) and it will automatically require all gems from Gemfile.